roachtest: django failed

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1838926&tab=buildLog) on [release-20.1@6a7ca722a135e21ad04daec3895535969ba5b02c](https://github.com/cockroachdb/cockroach/commits/6a7ca722a135e21ad04daec3895535969ba5b02c):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200331-1838926/django/run_1
	test_runner.go:800: test timed out (10h0m0s)

	django.go:192,django.go:212,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2016
		    main.registerDjango.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:189
		    main.registerDjango.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:212
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - cluster.RunL:
		  - context canceled

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=1838926&tab=artifacts#/django)
Related:
- #46490 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003240059_v20.1.0-beta.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003240059_v20.1.0-beta.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46375 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003200044_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003200044_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46321 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003181957_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003181957_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46171 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003161814_v19.2.5](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003161814_v19.2.5) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45393 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002240854_v20.1.0-beta.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002240854_v20.1.0-beta.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45299 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45063 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941)

- #44933 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002062122_v19.2.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002062122_v19.2.4)

- #44679 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #44573 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202001302015_v19.2.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202001302015_v19.2.3)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1841518&tab=buildLog) on [release-20.1@2f0eaf4891eec71d2f75a254f59b5c4af5e4c7b0](https://github.com/cockroachdb/cockroach/commits/2f0eaf4891eec71d2f75a254f59b5c4af5e4c7b0):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200401-1841518/django/run_1
	test_runner.go:800: test timed out (10h0m0s)

	django.go:192,django.go:212,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2016
		    main.registerDjango.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:189
		    main.registerDjango.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:212
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - cluster.RunL:
		  - context canceled

	cluster.go:1410,context.go:135,cluster.go:1399,test_runner.go:825: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-1841518-1585726225-34-n1cpu16 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM:
		  - 1: dead
		    main.glob..func13
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1129
		    main.wrap.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:272
		    github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra.(*Command).execute
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:766
		    github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra.(*Command).ExecuteC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:852
		    github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra.(*Command).Execute
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:800
		    main.main
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1793
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:203
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=1841518&tab=artifacts#/django)
Related:
- #46490 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003240059_v20.1.0-beta.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003240059_v20.1.0-beta.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46375 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003200044_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003200044_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46321 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003181957_v20.1.0-beta.3](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003181957_v20.1.0-beta.3) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #46171 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202003161814_v19.2.5](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202003161814_v19.2.5) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45393 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202002240854_v20.1.0-beta.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202002240854_v20.1.0-beta.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45299 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-45179](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-45179) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #45063 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-44941](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-44941)

- #44679 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1845326&tab=buildLog) on [release-20.1@7721f88440eb411445f7658130d42799610fc991](https://github.com/cockroachdb/cockroach/commits/7721f88440eb411445f7658130d42799610fc991):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200402-1845326/django/run_1
	test_runner.go:800: test timed out (10h0m0s)

	django.go:192,django.go:212,test_runner.go:753: error with attached stack trace:
		    main.execCmd
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:406
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2019
		    main.registerDjango.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:189
		    main.registerDjango.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:212
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: %s returned:
		    stderr:
		    %s
		    stdout:
		    %s
		    -- arg 1: <string>
		    -- arg 2: <string>
		    -- arg 3: <string>
		  - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1845326-1585812677-33-n1cpu16:1 -- cd /mnt/data1/django/tests && cat proxy_models.stdout returned:
		    stderr:
		    
		    stdout::
		  - context canceled

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=1845326&tab=artifacts#/django)
Related:
- #44679 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1851738&tab=buildLog) on [release-20.1@6dd7fc2278fc4c482abb769e7a1193d4ffaca967](https://github.com/cockroachdb/cockroach/commits/6dd7fc2278fc4c482abb769e7a1193d4ffaca967):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200404-1851738/django/run_1
	test_runner.go:800: test timed out (10h0m0s)

	django.go:192,django.go:212,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2016
		    main.registerDjango.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:189
		    main.registerDjango.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:212
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - cluster.RunL:
		  - context canceled

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=1851738&tab=artifacts#/django)
Related:
- #44679 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1853474&tab=buildLog) on [release-20.1@4deb223a5da181e7ba6857ac748f121a7a59d627](https://github.com/cockroachdb/cockroach/commits/4deb223a5da181e7ba6857ac748f121a7a59d627):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200405-1853474/django/run_1
	test_runner.go:800: test timed out (10h0m0s)

	django.go:192,django.go:212,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2016
		    main.registerDjango.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:189
		    main.registerDjango.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:212
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - cluster.RunL:
		  - context canceled

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=1853474&tab=artifacts#/django)
Related:
- #44679 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1857166&tab=buildLog) on [release-20.1@7699136a4861a6f6423e16ab7954fc4ac226bd6a](https://github.com/cockroachdb/cockroach/commits/7699136a4861a6f6423e16ab7954fc4ac226bd6a):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200407-1857166/django/run_1
	test_runner.go:800: test timed out (10h0m0s)

	django.go:192,django.go:212,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2016
		    main.registerDjango.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:189
		    main.registerDjango.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:212
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - cluster.RunL:
		  - context canceled

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=1857166&tab=artifacts#/django)
Related:
- #44679 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1860270&tab=buildLog) on [release-20.1@f5d82d1cdb75cfc3af9e35446d42c122050d9c9f](https://github.com/cockroachdb/cockroach/commits/f5d82d1cdb75cfc3af9e35446d42c122050d9c9f):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200408-1860270/django/run_1
	test_runner.go:800: test timed out (10h0m0s)

	django.go:192,django.go:212,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2026
		    main.registerDjango.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:189
		    main.registerDjango.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:212
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - cluster.RunL:
		  - context canceled

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=1860270&tab=artifacts#/django)
Related:
- #44679 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1862701&tab=buildLog) on [release-20.1@9d520536d4273264c58494222401b682d88d7edd](https://github.com/cockroachdb/cockroach/commits/9d520536d4273264c58494222401b682d88d7edd):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200409-1862701/django/run_1
	test_runner.go:800: test timed out (10h0m0s)

	django.go:192,django.go:212,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2026
		    main.registerDjango.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:189
		    main.registerDjango.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:212
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - cluster.RunL:
		  - context canceled

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=1862701&tab=artifacts#/django)
Related:
- #47282 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202004061746_v19.2.6](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202004061746_v19.2.6) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44679 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1866248&tab=buildLog) on [release-20.1@e593160235da38f0126546ea76374c796f2a194a](https://github.com/cockroachdb/cockroach/commits/e593160235da38f0126546ea76374c796f2a194a):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200410-1866248/django/run_1
	test_runner.go:800: test timed out (10h0m0s)

	django.go:192,django.go:212,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2026
		    main.registerDjango.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:189
		    main.registerDjango.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:212
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - cluster.RunL:
		  - context canceled

	cluster.go:1420,context.go:135,cluster.go:1409,test_runner.go:825: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-1866248-1586503466-33-n1cpu16 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM:
		  - 1: dead
		    main.glob..func13
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1129
		    main.wrap.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:272
		    github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra.(*Command).execute
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:766
		    github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra.(*Command).ExecuteC
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:852
		    github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra.(*Command).Execute
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:800
		    main.main
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1793
		    runtime.main
		    	/usr/local/go/src/runtime/proc.go:203
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=1866248&tab=artifacts#/django)
Related:
- #47282 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202004061746_v19.2.6](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202004061746_v19.2.6) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44679 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1868758&tab=buildLog) on [release-20.1@b1d5fdfc2801cb0b9ec8b0e1690e083901b1f5bc](https://github.com/cockroachdb/cockroach/commits/b1d5fdfc2801cb0b9ec8b0e1690e083901b1f5bc):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200411-1868758/django/run_1
	test_runner.go:800: test timed out (10h0m0s)

	django.go:192,django.go:212,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2026
		    main.registerDjango.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:189
		    main.registerDjango.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:212
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - cluster.RunL:
		  - context canceled

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=1868758&tab=artifacts#/django)
Related:
- #47282 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202004061746_v19.2.6](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202004061746_v19.2.6) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44679 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1878596&tab=buildLog) on [release-20.1@575a5bd7a8714464a74f52f337dc5af33558ebb0](https://github.com/cockroachdb/cockroach/commits/575a5bd7a8714464a74f52f337dc5af33558ebb0):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	test_runner.go:800: test timed out (10h0m0s)

	django.go:192,django.go:212,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2026
		    main.registerDjango.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:189
		    main.registerDjango.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:212
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - cluster.RunL:
		  - context canceled

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=1878596&tab=artifacts#/django)
Related:
- #47282 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202004061746_v19.2.6](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202004061746_v19.2.6) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44679 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1879172&tab=buildLog) on [release-20.1@778550737c285409cc1d9c6d96584fc985071c79](https://github.com/cockroachdb/cockroach/commits/778550737c285409cc1d9c6d96584fc985071c79):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	test_runner.go:800: test timed out (10h0m0s)

	django.go:192,django.go:212,test_runner.go:753: error with attached stack trace:
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2026
		    main.registerDjango.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:189
		    main.registerDjango.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/django.go:212
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:753
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - cluster.RunL:
		  - context canceled

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=1879172&tab=artifacts#/django)
Related:
- #47282 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202004061746_v19.2.6](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202004061746_v19.2.6) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44679 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2037503&tab=buildLog) on [release-20.1@988e55c9f1b0a1081bbfa918c1594ab312859a37](https://github.com/cockroachdb/cockroach/commits/988e55c9f1b0a1081bbfa918c1594ab312859a37):

```
		--- FAIL: file_storage.tests.FileFieldStorageTests.test_filefield_default (unexpected)
		--- FAIL: model_fields.test_imagefield.ImageFieldDimensionsFirstTests.test_field_save_and_delete_methods (unexpected)
		--- FAIL: expressions.tests.BasicExpressionsTests.test_in_subquery (unexpected)
		--- FAIL: model_fields.test_charfield.TestCharField.test_emoji (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_m2m_create_through (unexpected)
		--- FAIL: admin_views.tests.CustomModelAdminTest.test_custom_admin_site_login_template (unexpected)
		--- FAIL: m2m_through.tests.M2mThroughTests.test_custom_related_name_forward_empty_qs (unexpected)
		--- FAIL: model_fields.test_imagefield.ImageFieldTests.test_pickle (unexpected)
		--- FAIL: admin_views.tests.AdminViewListEditable.test_list_editable_ordering (unexpected)
		--- FAIL: queries.tests.Queries4Tests.test_ticket24525 (unexpected)
		--- FAIL: foreign_object.tests.MultiColumnFKTests.test_forward_in_lookup_filters_correctly (unexpected)
		--- FAIL: queries.tests.Queries1Tests.test_ticket2400 (unexpected)
		--- FAIL: aggregation_regress.tests.AggregationTests.test_having_group_by (unexpected)
		--- FAIL: defer.tests.DeferTests.test_only_with_select_related (unexpected)
		--- FAIL: expressions_window.tests.WindowFunctionTests.test_ntile (unexpected)
		--- FAIL: generic_relations.tests.GenericRelationsTests.test_add_bulk_false (unexpected)
		--- FAIL: from_db_value.tests.FromDBValueTest.test_simple_load (unexpected)
		--- FAIL: basic.tests.SelectOnSaveTests.test_select_on_save_lying_update (unexpected)
		--- FAIL: model_formsets.test_uuid.InlineFormsetTests.test_inlineformset_factory_nulls_default_pks_child_editable_pk (unexpected)
		--- FAIL: inline_formsets.tests.DeletionTests.test_save_new (unexpected)
		--- FAIL: get_or_create.tests.GetOrCreateTests.test_get_or_create_method_with_create (unexpected)
		--- FAIL: ordering.tests.OrderingTests.test_extra_ordering_with_table_name (unexpected)
		--- FAIL: generic_relations.tests.GenericRelationsTests.test_clear_after_prefetch (unexpected)
		--- FAIL: lookup.tests.LookupTests.test_lookup_date_as_str (unexpected)
		--- FAIL: m2m_through_regress.tests.ToFieldThroughTests.test_add_null_reverse (unexpected)
		--- FAIL: test_utils.tests.AssertNumQueriesContextManagerTests.test_with_client (unexpected)
		--- FAIL: syndication_tests.tests.SyndicationFeedTest.test_secure_urls (unexpected)
		--- FAIL: auth_tests.test_context_processors.AuthContextProcessorTests.test_perm_in_perms_attrs (unexpected)
		--- FAIL: expressions.tests.FTimeDeltaTests.test_date_comparison (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_m2m_create_inherited (unexpected)
		--- FAIL: expressions.tests.ExpressionOperatorTests.test_lefthand_power (unexpected)
		--- FAIL: admin_filters.tests.ListFiltersTests.test_allvaluesfieldlistfilter_custom_qs (unexpected)
		--- FAIL: filtered_relation.tests.FilteredRelationTests.test_with_generic_foreign_key (unexpected)
		--- FAIL: annotations.tests.NonAggregateAnnotationTestCase.test_combined_annotation_commutative (unexpected)
		--- FAIL: generic_relations.tests.GenericRelationsTests.test_set (unexpected)
		--- FAIL: model_fields.test_uuid.TestAsPrimaryKey.test_update_with_related_model_instance (unexpected)
		--- FAIL: admin_views.tests.AdminViewListEditable.test_post_submission (unexpected)
		--- FAIL: queries.tests.WeirdQuerysetSlicingTests.test_empty_sliced_subquery_exclude (unexpected)
		--- FAIL: model_fields.test_imagefield.ImageFieldTests.test_size_method (unexpected)
		--- FAIL: update_only_fields.tests.UpdateOnlyFieldsTests.test_update_fields_incorrect_params (unexpected)
		--- FAIL: annotations.tests.NonAggregateAnnotationTestCase.test_values_annotation (unexpected)
		--- FAIL: datetimes.tests.DateTimesTests.test_datetimes_returns_available_dates_for_given_scope_and_given_field (unexpected)
		--- FAIL: m2m_through.tests.M2mThroughReferentialTests.test_self_referential_empty_qs (unexpected)
		--- FAIL: serializers.test_yaml.YamlSerializerTestCase.test_pkless_serialized_strings (unexpected)
		--- FAIL: multiple_database.tests.RouteForWriteTestCase.test_reverse_m2m_remove (unexpected)
		--- FAIL: queries.tests.QuerySetSupportsPythonIdioms.test_slicing_with_tests_is_not_lazy (unexpected)
		--- FAIL: foreign_object.tests.MultiColumnFKTests.test_reverse_query_filters_correctly (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2037503&tab=artifacts#/django)
Related:
- #50474 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006220937_v19.2.8](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006220937_v19.2.8) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50040 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006091546_v20.1.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006091546_v20.1.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2050364&tab=buildLog) on [release-20.1@e7779ad82559c4a4b8f0fd504f64e18dbd2ade98](https://github.com/cockroachdb/cockroach/commits/e7779ad82559c4a4b8f0fd504f64e18dbd2ade98):

```
		--- FAIL: generic_views.test_detail.DetailViewTest.test_verbose_name (unexpected)
		--- FAIL: migrations.test_loader.LoaderTests.test_check_consistent_history_squashed (unexpected)
		--- FAIL: serializers.test_yaml.YamlSerializerTestCase.test_serialize_to_stream (unexpected)
		--- FAIL: admin_docs.test_views.AdminDocViewWithMultipleEngines.test_view_detail (unexpected)
		--- FAIL: admin_filters.tests.ListFiltersTests.test_lookup_with_non_string_value (unexpected)
		--- FAIL: db_functions.math.test_degrees.DegreesTests.test_integer (unexpected)
		--- FAIL: queries.tests.ValuesQuerysetTests.test_named_values_list_expression_with_default_alias (unexpected)
		--- FAIL: select_related_onetoone.tests.ReverseSelectRelatedTestCase.test_multiple_subclass (unexpected)
		--- FAIL: expressions_window.tests.WindowFunctionTests.test_rank (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_alter_fk_non_fk (unexpected)
		--- FAIL: admin_views.tests.CSSTest.test_field_prefix_css_classes (unexpected)
		--- FAIL: admin_views.tests.CustomModelAdminTest.test_custom_admin_site_password_change_done_template (unexpected)
		--- FAIL: m2m_through_regress.tests.M2MThroughTestCase.test_retrieve_forward_m2m_items_via_custom_id_intermediary (unexpected)
		--- FAIL: many_to_one.tests.ManyToOneTests.test_reverse_foreign_key_instance_to_field_caching (unexpected)
		--- FAIL: fixtures.tests.FixtureLoadingTests.test_dumpdata_with_excludes (unexpected)
		--- FAIL: generic_relations_regress.tests.GenericRelationTests.test_reverse_relation_pk (unexpected)
		--- FAIL: model_formsets.tests.ModelFormsetTest.test_inline_formsets_with_multi_table_inheritance (unexpected)
		--- FAIL: prefetch_related.tests.CustomPrefetchTests.test_to_attr_cached_property (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_alter_field_reloads_state_on_fk_with_to_field_target_type_change (unexpected)
		--- FAIL: bulk_create.tests.BulkCreateTests.test_large_batch_mixed (unexpected)
		--- FAIL: model_fields.test_booleanfield.BooleanFieldTests.test_select_related (unexpected)
		--- FAIL: custom_columns.tests.CustomColumnsTests.test_author_get_attributes (unexpected)
		--- FAIL: timezones.tests.NewDatabaseTests.test_query_filter_with_pytz_timezones (unexpected)
		--- FAIL: auth_tests.test_auth_backends.PermissionDeniedBackendTest.test_has_perm_denied (unexpected)
		--- FAIL: backends.base.test_base.ExecuteWrapperTests.test_wrapper_invoked_many (unexpected)
		--- FAIL: auth_tests.test_management.ChangepasswordManagementCommandTestCase.test_password_validation (unexpected)
		--- FAIL: admin_views.tests.AdminViewBasicTest.test_named_group_field_choices_change_list (unexpected)
		--- FAIL: constraints.tests.CheckConstraintTests.test_database_constraint (unexpected)
		--- FAIL: queries.test_qs_combinators.QuerySetSetOperationTests.test_simple_difference (unexpected)
		--- FAIL: auth_tests.test_forms.AdminPasswordChangeFormTest.test_missing_passwords (unexpected)
		--- FAIL: fixtures_regress.tests.TestFixtures.test_loaddata_raises_error_when_fixture_has_invalid_foreign_key (unexpected)
		--- FAIL: expressions.tests.ExpressionOperatorTests.test_lefthand_bitwise_left_shift_operator (unexpected)
		--- FAIL: model_fields.test_imagefield.ImageFieldUsingFileTests.test_field_save_and_delete_methods (unexpected)
		--- FAIL: proxy_models.tests.ProxyModelTests.test_filter_proxy_relation_reverse (unexpected)
		--- FAIL: queries.test_qs_combinators.QuerySetSetOperationTests.test_count_difference (unexpected)
		--- FAIL: admin_utils.test_logentry.LogEntryTests.test_log_action (unexpected)
		--- FAIL: admin_views.tests.CustomModelAdminTest.test_pwd_change_custom_template (unexpected)
		--- FAIL: update_only_fields.tests.UpdateOnlyFieldsTests.test_update_fields_inheritance (unexpected)
		--- FAIL: order_with_respect_to.tests.OrderWithRespectToBaseTests.test_default_to_insertion_order (unexpected)
		--- FAIL: admin_utils.test_logentry.LogEntryTests.test_logentry_unicode (unexpected)
		--- FAIL: auth_tests.test_auth_backends.CustomPermissionsUserModelBackendTest.test_inactive_has_no_permissions (unexpected)
		--- FAIL: db_functions.math.test_round.RoundTests.test_integer (unexpected)
		--- FAIL: m2m_through.tests.M2mThroughTests.test_remove_on_reverse_m2m_with_intermediate_model (unexpected)
		--- FAIL: basic.tests.ModelTest.test_multiple_objects_max_num_fetched (unexpected)
		--- FAIL: db_functions.datetime.test_extract_trunc.DateFunctionTests.test_extract_minute_func (unexpected)
		--- FAIL: multiple_database.tests.SignalTests.test_database_arg_save_and_delete (unexpected)
		--- FAIL: admin_views.test_actions.AdminActionsTest.test_model_without_action_still_has_jquery (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2050364&tab=artifacts#/django)
Related:
- #50793 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292135_v19.2.9](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292135_v19.2.9) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2054042&tab=buildLog) on [release-20.1@9c486c55b8a92cf2f1e441458b7a8c7ee2a55679](https://github.com/cockroachdb/cockroach/commits/9c486c55b8a92cf2f1e441458b7a8c7ee2a55679):

```
		--- FAIL: timezones.tests.LegacyDatabaseTests.test_query_datetime_lookups (unexpected)
		--- FAIL: generic_views.test_dates.ArchiveIndexViewTests.test_archive_view_template_suffix (unexpected)
		--- FAIL: select_related.tests.SelectRelatedTests.test_chaining (unexpected)
		--- FAIL: select_related_regress.tests.SelectRelatedRegressTests.test_regression_10733 (unexpected)
		--- FAIL: admin_docs.test_views.AdminDocViewWithMultipleEngines.test_no_sites_framework (unexpected)
		--- FAIL: multiple_database.tests.RouteForWriteTestCase.test_reverse_fk_update (unexpected)
		--- FAIL: auth_tests.test_management.CreatesuperuserManagementCommandTestCase.test_environment_variable_non_interactive (unexpected)
		--- FAIL: expressions.tests.ExpressionOperatorTests.test_lefthand_division (unexpected)
		--- FAIL: expressions.tests.FieldTransformTests.test_multiple_transforms_in_values (unexpected)
		--- FAIL: many_to_many.tests.ManyToManyTests.test_fast_add_ignore_conflicts (unexpected)
		--- FAIL: managers_regress.tests.ManagersRegressionTests.test_field_can_be_called_exact (unexpected)
		--- FAIL: model_forms.tests.ModelFormBasicTests.test_subset_fields (unexpected)
		--- FAIL: admin_changelist.tests.ChangeListTests.test_distinct_for_many_to_many_at_second_level_in_search_fields (unexpected)
		--- FAIL: auth_tests.test_basic.BasicTestCase.test_unicode_username (unexpected)
		--- FAIL: model_fields.test_autofield.AutoFieldTests.test_types (unexpected)
		--- FAIL: basic.tests.ModelTest.test_microsecond_precision (unexpected)
		--- FAIL: many_to_one.tests.ManyToOneTests.test_fk_instantiation_outside_model (unexpected)
		--- FAIL: multiple_database.tests.RouteForWriteTestCase.test_reverse_m2m_update (unexpected)
		--- FAIL: select_related_onetoone.tests.ReverseSelectRelatedTestCase.test_follow_next_level (unexpected)
		--- FAIL: fixtures_regress.tests.TestFixtures.test_duplicate_pk (unexpected)
		--- FAIL: generic_views.test_dates.ArchiveIndexViewTests.test_no_duplicate_query (unexpected)
		--- FAIL: bulk_create.tests.BulkCreateTests.test_empty_model (unexpected)
		--- FAIL: db_functions.math.test_sqrt.SqrtTests.test_null (unexpected)
		--- FAIL: generic_views.test_detail.DetailViewTest.test_detail_by_custom_pk (unexpected)
		--- FAIL: admin_views.tests.AdminUserMessageTest.test_message_warning (unexpected)
		--- FAIL: model_fields.test_uuid.TestAsPrimaryKey.test_uuid_pk_on_save (unexpected)
		--- FAIL: admin_views.tests.AdminViewPermissionsTest.test_overriding_has_module_permission (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_m2m_custom (unexpected)
		--- FAIL: custom_columns.tests.CustomColumnsTests.test_get_author_m2m_relation (unexpected)
		--- FAIL: syndication_tests.tests.SyndicationFeedTest.test_item_link_error (unexpected)
		--- FAIL: bulk_create.tests.BulkCreateTests.test_set_pk_and_query_efficiency (unexpected)
		--- FAIL: introspection.tests.IntrospectionTests.test_sequence_list (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: db_functions.math.test_ln.LnTests.test_null (unexpected)
		--- FAIL: prefetch_related.test_uuid.UUIDPrefetchRelated.test_prefetch_related_to_uuid_model (unexpected)
		--- FAIL: admin_views.tests.AdminCustomQuerysetTest.test_edit_model_modeladmin_defer_qs (unexpected)
		--- FAIL: auth_tests.test_views.PasswordResetTest.test_confirm_redirect_default (unexpected)
		--- FAIL: admin_views.tests.AdminViewBasicTest.test_edit_save_as (unexpected)
		--- FAIL: many_to_many.tests.ManyToManyTests.test_delete (unexpected)
		--- FAIL: custom_pk.tests.BasicCustomPKTests.test_querysets_related_name (unexpected)
		--- FAIL: generic_relations.tests.ProxyRelatedModelTest.test_generic_relation_set (unexpected)
		--- FAIL: known_related_objects.tests.ExistingRelatedInstancesTests.test_foreign_key_prefetch_related (unexpected)
		--- FAIL: model_forms.test_modelchoicefield.ModelChoiceFieldTests.test_choices_bool_empty_label (unexpected)
		--- FAIL: generic_views.test_edit.CreateViewTests.test_create_without_redirect (unexpected)
		--- FAIL: admin_filters.tests.ListFiltersTests.test_booleanfieldlistfilter_tuple (unexpected)
		--- FAIL: db_functions.comparison.test_greatest.GreatestTests.test_related_field (unexpected)
		--- FAIL: select_related_regress.tests.SelectRelatedRegressTests.test_regression_8106 (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2054042&tab=artifacts#/django)
Related:
- #50793 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292135_v19.2.9](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292135_v19.2.9) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2058112&tab=buildLog) on [release-20.1@11d292c8c888506721da28eb6bc806aa188b53c2](https://github.com/cockroachdb/cockroach/commits/11d292c8c888506721da28eb6bc806aa188b53c2):

```
		--- FAIL: admin_inlines.tests.TestInlinePermissions.test_inline_add_m2m_view_only_perm (unexpected)
		--- FAIL: auth_tests.test_auth_backends.CustomPermissionsUserModelBackendTest.test_has_perm (unexpected)
		--- FAIL: model_formsets.tests.TestModelFormsetOverridesTroughFormMeta.test_inlineformset_factory_field_class_overrides (unexpected)
		--- FAIL: admin_utils.tests.NestedObjectsTests.test_on_delete_do_nothing (unexpected)
		--- FAIL: db_functions.comparison.test_cast.CastTests.test_cast_from_db_datetime_to_date_group_by (unexpected)
		--- FAIL: auth_tests.test_forms.AuthenticationFormTest.test_login_failed (unexpected)
		--- FAIL: auth_tests.test_forms.PasswordResetFormTest.test_inactive_user (unexpected)
		--- FAIL: queryset_pickle.tests.PickleabilityTestCase.test_unsupported_unpickle (unexpected)
		--- FAIL: model_forms.test_modelchoicefield.ModelChoiceFieldTests.test_choices_bool_empty_label (unexpected)
		--- FAIL: db_functions.datetime.test_extract_trunc.DateFunctionWithTimeZoneTests.test_trunc_timezone_applied_before_truncation (unexpected)
		--- FAIL: queries.tests.Queries2Tests.test_ticket4289 (unexpected)
		--- FAIL: aggregation_regress.tests.AggregationTests.test_negated_aggregation (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_m2m_inherited (unexpected)
		--- FAIL: admin_filters.tests.ListFiltersTests.test_lookup_with_dynamic_value (unexpected)
		--- FAIL: model_fields.test_imagefield.ImageFieldOneDimensionTests.test_create (unexpected)
		--- FAIL: aggregation.tests.AggregateTestCase.test_filtering (unexpected)
		--- FAIL: auth_tests.test_management.CreatesuperuserManagementCommandTestCase.test_validate_password_against_username (unexpected)
		--- FAIL: queries.test_bulk_update.BulkUpdateTests.test_only_concrete_fields_allowed (unexpected)
		--- FAIL: test_client.tests.ClientTest.test_relative_redirect_no_trailing_slash (unexpected)
		--- FAIL: defer.tests.BigChildDeferTests.test_defer_subclass_both (unexpected)
		--- FAIL: expressions.tests.FTimeDeltaTests.test_exclude (unexpected)
		--- FAIL: generic_relations.tests.GenericRelationsTests.test_prefetch_related_different_content_types (unexpected)
		--- FAIL: db_functions.tests.FunctionTests.test_nested_function_ordering (unexpected)
		--- FAIL: queries.tests.Queries4Tests.test_ticket7095 (unexpected)
		--- FAIL: admin_inlines.tests.TestInline.test_custom_min_num (unexpected)
		--- FAIL: admin_views.test_actions.AdminActionsTest.test_popup_template_response_on_delete (unexpected)
		--- FAIL: admin_views.tests.AdminViewListEditable.test_list_editable_popup (unexpected)
		--- FAIL: aggregation_regress.tests.AggregationTests.test_aggregation_with_generic_reverse_relation (unexpected)
		--- FAIL: admin_views.tests.AdminCustomSaveRelatedTests.test_should_be_able_to_edit_related_objects_on_change_view (unexpected)
		--- FAIL: backends.tests.FkConstraintsTests.test_check_constraints (unexpected)
		--- FAIL: db_functions.text.test_lower.LowerTests.test_transform (unexpected)
		--- FAIL: managers_regress.tests.ManagersRegressionTests.test_managers (unexpected)
		--- FAIL: queries.tests.ExcludeTests.test_ticket14511 (unexpected)
		--- FAIL: get_or_create.tests.UpdateOrCreateTests.test_with_pk_property (unexpected)
		--- FAIL: many_to_many.tests.ManyToManyTests.test_set (unexpected)
		--- FAIL: prefetch_related.tests.DirectPrefetchedObjectCacheReuseTests.test_remove_clears_prefetched_objects (unexpected)
		--- FAIL: aggregation_regress.tests.AggregationTests.test_quoting_aggregate_order_by (unexpected)
		--- FAIL: foreign_object.tests.MultiColumnFKTests.test_m2m_through_forward_returns_valid_members (unexpected)
		--- FAIL: queries.tests.Queries1Tests.test_ticket7096 (unexpected)
		--- FAIL: update.tests.AdvancedTests.test_update_annotated_queryset (unexpected)
		--- FAIL: generic_views.test_dates.ArchiveIndexViewTests.test_archive_view (unexpected)
		--- FAIL: sites_tests.tests.SitesFrameworkTests.test_site_natural_key (unexpected)
		--- FAIL: admin_views.tests.AdminViewLogoutTests.test_logout (unexpected)
		--- FAIL: generic_relations.tests.GenericRelationsTests.test_queries_across_generic_relations (unexpected)
		--- FAIL: model_fields.test_autofield.BigAutoFieldTests.test_documented_range (unexpected)
		--- FAIL: queries.tests.Ticket14056Tests.test_ticket_14056 (unexpected)
		--- FAIL: admin_changelist.tests.ChangeListTests.test_select_related_as_empty_tuple (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2058112&tab=artifacts#/django)
Related:
- #50793 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292135_v19.2.9](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292135_v19.2.9) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2061284&tab=buildLog) on [release-20.1@039506a899dbc6b29516932c671c1c672bb73ae7](https://github.com/cockroachdb/cockroach/commits/039506a899dbc6b29516932c671c1c672bb73ae7):

```
		--- FAIL: db_functions.math.test_round.RoundTests.test_null (unexpected)
		--- FAIL: one_to_one.tests.OneToOneTests.test_cached_relation_invalidated_on_save (unexpected)
		--- FAIL: foreign_object.tests.MultiColumnFKTests.test_double_nested_query (unexpected)
		--- FAIL: multiple_database.tests.RouteForWriteTestCase.test_reverse_m2m_get_or_create (unexpected)
		--- FAIL: auth_tests.test_remote_user.RemoteUserCustomTest.test_known_user (unexpected)
		--- FAIL: model_fields.test_binaryfield.BinaryFieldTests.test_filter_memoryview (unexpected)
		--- FAIL: many_to_many.tests.ManyToManyTests.test_custom_default_manager_exists_count (unexpected)
		--- FAIL: many_to_one_null.tests.ManyToOneNullTests.test_remove_from_wrong_set (unexpected)
		--- FAIL: model_fields.test_imagefield.TwoImageFieldTests.test_create (unexpected)
		--- FAIL: admin_docs.test_views.AdminDocViewTests.test_view_detail_as_method (unexpected)
		--- FAIL: m2m_through.tests.M2mThroughTests.test_get_or_create_on_m2m_with_intermediate_model_value_required_fails (unexpected)
		--- FAIL: expressions.tests.BasicExpressionsTests.test_filtering_on_q_that_is_boolean (unexpected)
		--- FAIL: prefetch_related.test_uuid.UUIDPrefetchRelatedLookups.test_from_integer_pk_lookup_uuid_pk_integer_pk (unexpected)
		--- FAIL: queries.tests.Queries2Tests.test_ticket7759 (unexpected)
		--- FAIL: auth_tests.test_templates.AuthTemplateTests.test_PasswordResetConfirmView_valid_token (unexpected)
		--- FAIL: cache.tests.FileBasedCacheTests.test_cache_write_for_model_instance_with_deferred (unexpected)
		--- FAIL: foreign_object.tests.MultiColumnFKTests.test_prefetch_foreignkey_forward_works (unexpected)
		--- FAIL: queries.test_iterator.QuerySetIteratorTests.test_no_chunked_reads (unexpected)
		--- FAIL: raw_query.tests.RawQueryTests.test_result_caching (unexpected)
		--- FAIL: test_client.tests.ClientTest.test_relative_redirect (unexpected)
		--- FAIL: timezones.tests.LegacyDatabaseTests.test_raw_sql (unexpected)
		--- FAIL: queries.tests.Queries1Tests.test_ticket_10790_5 (unexpected)
		--- FAIL: queries.test_qs_combinators.QuerySetSetOperationTests.test_difference_with_empty_qs (unexpected)
		--- FAIL: datatypes.tests.DataTypesTestCase.test_time_field (unexpected)
		--- FAIL: model_forms.test_modelchoicefield.ModelChoiceFieldTests.test_choices_freshness (unexpected)
		--- FAIL: queries.tests.ExcludeTests.test_ticket14511 (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_m2m_create_through_custom (unexpected)
		--- FAIL: expressions_case.tests.CaseDocumentationExamples.test_conditional_update_example (unexpected)
		--- FAIL: admin_views.test_actions.AdminActionsTest.test_popup_template_response_on_add (unexpected)
		--- FAIL: custom_managers.tests.CustomManagersRegressTestCase.test_queryset_with_custom_init (unexpected)
		--- FAIL: db_functions.math.test_floor.FloorTests.test_integer (unexpected)
		--- FAIL: modeladmin.tests.ModelAdminTests.test_foreign_key_as_radio_field (unexpected)
		--- FAIL: queries.tests.Queries4Tests.test_ticket15316_one2one_exclude_true (unexpected)
		--- FAIL: auth_tests.test_remote_user.AllowAllUsersRemoteUserBackendTest.test_no_remote_user (unexpected)
		--- FAIL: m2m_through.tests.M2mThroughReferentialTests.test_self_referential_non_symmetrical_first_side (unexpected)
		--- FAIL: db_functions.comparison.test_least.LeastTests.test_coalesce_workaround (unexpected)
		--- FAIL: m2m_through.tests.M2mThroughTests.test_update_or_create_on_m2m_with_intermediate_model_value_required (unexpected)
		--- FAIL: generic_views.test_detail.DetailViewTest.test_verbose_name (unexpected)
		--- FAIL: aggregation_regress.tests.AggregationTests.test_aggregate_duplicate_columns (unexpected)
		--- FAIL: view_tests.tests.test_i18n.SetLanguageTests.test_setlang (unexpected)
		--- FAIL: many_to_one.tests.ManyToOneTests.test_delete (unexpected)
		--- FAIL: filtered_relation.tests.FilteredRelationTests.test_with_join_and_complex_condition (unexpected)
		--- FAIL: custom_managers.tests.CustomManagerTests.test_slow_removal_through_specified_gfk_related_manager (unexpected)
		--- FAIL: model_forms.tests.ModelMultipleChoiceFieldTests.test_show_hidden_initial_changed_queries_efficiently (unexpected)
		--- FAIL: auth_tests.test_management.ChangepasswordManagementCommandTestCase.test_that_changepassword_command_works_with_nonascii_output (unexpected)
		--- FAIL: basic.tests.ModelTest.test_hash_function (unexpected)
		--- FAIL: auth_tests.test_views.ChangePasswordTest.test_password_change_redirect_custom (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2061284&tab=artifacts#/django)
Related:
- #50793 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292135_v19.2.9](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292135_v19.2.9) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #44477 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2068320&tab=buildLog) on [release-20.1@7a5876b419b7b0b68b8252f141cc013d8f39b70a](https://github.com/cockroachdb/cockroach/commits/7a5876b419b7b0b68b8252f141cc013d8f39b70a):

```
		--- FAIL: migrations.test_executor.ExecutorTests.test_alter_id_type_with_fk (unexpected)
		--- FAIL: ordering.tests.OrderingTests.test_order_by_pk (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_stealth_table_name_filter_option (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_alter_not_unique_field_to_primary_key (unexpected)
		--- FAIL: aggregation_regress.tests.AggregationTests.test_ticket_11293 (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_alter_text_field_to_datetime_field (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_alter_field_reloads_state_on_fk_with_to_field_target_type_change (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_table_option (unexpected)
		--- FAIL: ordering.tests.OrderingTests.test_order_by_fk_attname (unexpected)
		--- FAIL: expressions.tests.ExpressionOperatorTests.test_lefthand_division (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_special_column_name_introspection (unexpected)
		--- FAIL: introspection.tests.IntrospectionTests.test_get_table_description_col_lengths (unexpected)
		--- FAIL: backends.tests.FkConstraintsTests.test_disable_constraint_checks_manually (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_alter_fk_non_fk (unexpected)
		--- FAIL: queries.tests.Queries1Tests.test_ticket9411 (unexpected)
		--- FAIL: queries.tests.SubqueryTests.test_slice_subquery_and_query (unexpected)
		--- FAIL: defer_regress.tests.DeferRegressionTest.test_ticket_23270 (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_rename_field_reloads_state_on_fk_target_changes (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_alter_field_reloads_state_on_fk_with_to_field_target_changes (unexpected)
		--- FAIL: syndication_tests.tests.SyndicationFeedTest.test_rss2_feed (unexpected)
		--- FAIL: db_functions.text.test_sha224.SHA224Tests.test_transform (unexpected)
		--- FAIL: aggregation.tests.AggregateTestCase.test_expression_on_aggregation (unexpected)
		--- FAIL: expressions_case.tests.CaseExpressionTests.test_update_fk (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_rename (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_alter_field_reloads_state_on_fk_with_to_field_related_name_target_type_change (unexpected)
		--- FAIL: syndication_tests.tests.SyndicationFeedTest.test_rss091_feed (unexpected)
		--- FAIL: db_functions.text.test_sha384.SHA384Tests.test_basic (unexpected)
		--- FAIL: aggregation.tests.AggregateTestCase.test_combine_different_types (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_alter_order_with_respect_to (unexpected)
		--- FAIL: annotations.tests.NonAggregateAnnotationTestCase.test_mixed_type_annotation_numbers (unexpected)
		--- FAIL: aggregation.test_filter_argument.FilteredAggregateTests.test_filtered_numerical_aggregates (unexpected)
		--- FAIL: #16536/#17676.Introspection of column names consist/start with digits (unexpected)
		--- FAIL: expressions.tests.FTimeDeltaTests.test_datetime_subtraction_microseconds (unexpected)
		--- FAIL: aggregation_regress.tests.AggregationTests.test_stddev (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_char_field_with_db_index_to_fk (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_alter_field_reloads_state_on_fk_target_changes (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_alter_autofield_pk_to_smallautofield_pk_sequence_owner (unexpected)
		--- FAIL: aggregation_regress.tests.AggregationTests.test_more_more (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_text_field_with_db_index_to_fk (unexpected)
		--- FAIL: queries.tests.Ticket14056Tests.test_ticket_14056 (unexpected)
		--- FAIL: timezones.tests.LegacyDatabaseTests.test_cursor_execute_accepts_naive_datetime (unexpected)
		--- FAIL: #14305.By default the command generates models with `Meta.managed = False` (unexpected)
		--- FAIL: aggregation.tests.AggregateTestCase.test_add_implementation (unexpected)
		--- FAIL: syndication_tests.tests.SyndicationFeedTest.test_latest_post_date (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_alter_textual_field_keep_null_status (unexpected)
		--- FAIL: generic_relations_regress.tests.GenericRelationTests.test_annotate (unexpected)
		--- FAIL: model_options.test_tablespaces.TablespacesTests.test_tablespace_for_many_to_many_field (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2068320&tab=artifacts#/django)
Related:
- #51049 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50793 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292135_v19.2.9](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292135_v19.2.9) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2072472&tab=buildLog) on [release-20.1@a565ade7001ff5844581fec834b542fd0532c180](https://github.com/cockroachdb/cockroach/commits/a565ade7001ff5844581fec834b542fd0532c180):

```
		--- FAIL: migrations.test_operations.OperationTests.test_alter_field_pk_fk (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_alter_text_field_to_datetime_field (unexpected)
		--- FAIL: queries.test_bulk_update.BulkUpdateTests.test_inherited_fields (unexpected)
		--- FAIL: syndication_tests.tests.SyndicationFeedTest.test_rss091_feed (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_alter_field_reloads_state_on_fk_target_changes (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_alter_order_with_respect_to (unexpected)
		--- FAIL: db_functions.text.test_sha224.SHA224Tests.test_basic (unexpected)
		--- FAIL: queries.tests.NullInExcludeTest.test_col_not_in_list_containing_null (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: ordering.tests.OrderingTests.test_order_by_pk (unexpected)
		--- FAIL: aggregation_regress.tests.AggregationTests.test_more_more_more (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_primary_key (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_alter_text_field_to_time_field (unexpected)
		--- FAIL: defer_regress.tests.DeferRegressionTest.test_ticket_23270 (unexpected)
		--- FAIL: generic_relations_regress.tests.GenericRelationTests.test_annotate (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_table_name_introspection (unexpected)
		--- FAIL: queries.tests.SubqueryTests.test_slice_subquery_and_query (unexpected)
		--- FAIL: syndication_tests.tests.SyndicationFeedTest.test_latest_post_date (unexpected)
		--- FAIL: ordering.tests.OrderingTests.test_order_by_fk_attname (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_alter_text_field_to_date_field (unexpected)
		--- FAIL: timezones.tests.LegacyDatabaseTests.test_cursor_execute_accepts_naive_datetime (unexpected)
		--- FAIL: #14305.By default the command generates models with `Meta.managed = False` (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_alter_auto_field_to_char_field (unexpected)
		--- FAIL: queries.tests.Queries1Tests.test_ticket9411 (unexpected)
		--- FAIL: migrations.test_executor.ExecutorTests.test_alter_id_type_with_fk (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_rename_field_reloads_state_on_fk_target_changes (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_table_option (unexpected)
		--- FAIL: queries.test_explain.ExplainTests.test_unknown_format (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_special_column_name_introspection (unexpected)
		--- FAIL: queries.test_bulk_update.BulkUpdateNoteTests.test_multiple_fields (unexpected)
		--- FAIL: aggregation.tests.AggregateTestCase.test_expression_on_aggregation (unexpected)
		--- FAIL: aggregation.tests.AggregateTestCase.test_combine_different_types (unexpected)
		--- FAIL: backends.tests.FkConstraintsTests.test_disable_constraint_checks_manually (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_alter_field_reloads_state_on_fk_with_to_field_target_type_change (unexpected)
		--- FAIL: schema.tests.SchemaTests.test_alter_int_pk_to_int_unique (unexpected)
		--- FAIL: queries.tests.RelatedLookupTypeTests.test_values_queryset_lookup (unexpected)
		--- FAIL: expressions_case.tests.CaseExpressionTests.test_update_fk (unexpected)
		--- FAIL: introspection.tests.IntrospectionTests.test_sequence_list (unexpected)
		--- FAIL: aggregation_regress.tests.AggregationTests.test_stddev (unexpected)
		--- FAIL: model_inheritance_regress.tests.ModelInheritanceTest.test_inheritance_values_joins (unexpected)
		--- FAIL: migrations.test_operations.OperationTests.test_alter_fk_non_fk (unexpected)
		--- FAIL: syndication_tests.tests.SyndicationFeedTest.test_rss2_feed (unexpected)
		--- FAIL: queries.tests.ExcludeTest17600.test_only_orders_with_all_items_having_status_1 (unexpected)
		--- FAIL: backends.tests.FkConstraintsTests.test_check_constraints (unexpected)
		--- FAIL: db_functions.text.test_sha224.SHA224Tests.test_transform (unexpected)
		--- FAIL: introspection.tests.IntrospectionTests.test_table_names_with_views (unexpected)
		--- FAIL: aggregation.test_filter_argument.FilteredAggregateTests.test_filtered_numerical_aggregates (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2072472&tab=artifacts#/django)
Related:
- #51113 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007071743_v20.2.0-alpha.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007071743_v20.2.0-alpha.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #51051 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #51049 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #50793 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202006292135_v19.2.9](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202006292135_v19.2.9) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2081789&tab=buildLog) on [release-20.1@a36656a9f66ef1bd61873cfbaed6d1ea8e4b75a8](https://github.com/cockroachdb/cockroach/commits/a36656a9f66ef1bd61873cfbaed6d1ea8e4b75a8):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:214,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:757: 
		Tests run on Cockroach v20.1.3-43-ga36656a
		Tests run against django 3.0.6
		8023 Total Tests Run
		8013 tests passed
		10 tests failed
		518 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		10 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_stealth_table_name_filter_option (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_special_column_name_introspection (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_digits_column_name_introspection (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_attribute_name_not_python_keyword (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_field_types (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_table_option (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_unique_together_meta (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_table_name_introspection (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_managed_models (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_number_field_types (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2081789&tab=artifacts#/django)
Related:
- #51320 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007081918_v20.2.0-alpha.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007081918_v20.2.0-alpha.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2123176&tab=buildLog) on [release-20.1@b42f9f14ef5631c284cb4400f02e6e33a46f3cea](https://github.com/cockroachdb/cockroach/commits/b42f9f14ef5631c284cb4400f02e6e33a46f3cea):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:214,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:754: 
		Tests run on Cockroach v20.1.3-90-gb42f9f1
		Tests run against django 3.0.6
		8023 Total Tests Run
		8013 tests passed
		10 tests failed
		518 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		10 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_attribute_name_not_python_keyword (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_table_option (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_field_types (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_special_column_name_introspection (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_digits_column_name_introspection (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_number_field_types (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_managed_models (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_unique_together_meta (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_table_name_introspection (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_stealth_table_name_filter_option (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2123176&tab=artifacts#/django)
Related:
- #51962 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007271721_v20.1.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007271721_v20.1.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #51710 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007220233_v20.2.0-alpha.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007220233_v20.2.0-alpha.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #51369 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #51320 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007081918_v20.2.0-alpha.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007081918_v20.2.0-alpha.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2130485&tab=buildLog) on [release-20.1@12049d3fe3650660e1b6abf1e522d9bb016acb88](https://github.com/cockroachdb/cockroach/commits/12049d3fe3650660e1b6abf1e522d9bb016acb88):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:214,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:754: 
		Tests run on Cockroach v20.1.3-97-g12049d3
		Tests run against django 3.0.6
		8010 Total Tests Run
		8002 tests passed
		8 tests failed
		518 tests skipped
		13 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_managed_models (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_attribute_name_not_python_keyword (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_table_option (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_stealth_table_name_filter_option (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_number_field_types (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_table_name_introspection (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_special_column_name_introspection (unexpected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_unique_together_meta (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2130485&tab=artifacts#/django)
Related:
- #52109 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007292238_v20.1.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007292238_v20.1.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #52088 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007291551_v20.1.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007291551_v20.1.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #51962 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007271721_v20.1.4](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007271721_v20.1.4) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #51710 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007220233_v20.2.0-alpha.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007220233_v20.2.0-alpha.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #51369 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #51320 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202007081918_v20.2.0-alpha.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202007081918_v20.2.0-alpha.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/django.go
+++ b/pkg/cmd/roachtest/django.go
@@ -16,7 +16,7 @@ import (
 	"regexp"
 )
 
-var djangoReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)$`)
+var djangoReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)(\.(?P<point>\d+))?$`)
 var djangoCockroachDBReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)$`)
 
 var djangoSupportedTag = "3.0.6"

--- a/pkg/cmd/roachtest/sqlalchemy_blocklist.go
+++ b/pkg/cmd/roachtest/sqlalchemy_blocklist.go
@@ -18,7 +18,9 @@ var sqlAlchemyBlocklists = blocklistsForVersion{
 	{"v20.2", "sqlAlchemyBlocklist20_2", sqlAlchemyBlocklist20_2, "sqlAlchemyIgnoreList20_2", sqlAlchemyIgnoreList20_2},
 }
 
-var sqlAlchemyBlocklist20_2 = blocklist{}
+var sqlAlchemyBlocklist20_2 = blocklist{
+	"test/dialect/test_suite.py::ComponentReflectionTest_cockroachdb+psycopg2_9_5_0::test_get_foreign_keys": "52356",
+}
 
 var sqlAlchemyBlocklist20_1 = blocklist{
 	"test/dialect/test_suite.py::ExpandingBoundInTest_cockroachdb+psycopg2_9_5_0::test_null_in_empty_set_is_false": "41596",

